src/Compilers/Test/Utilities/CSharp/MockCSharpCompiler.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Test.Utilities
 {
     internal class MockCSharpCompiler : CSharpCompiler
     {
-        private readonly ImmutableArray<DiagnosticAnalyzer> _analyzers;
+        protected readonly ImmutableArray<DiagnosticAnalyzer> _analyzers;
```

Comment:
I think our naming convention is different for `protected` variables, but if this is only `protected` for testing purposes, you may want to put a comment and leave it as-is.

Reply:
Talked to Chuck, who said he uses either underscore or lower-case for protected.
I think the underscore is acceptable. 
Note: This whole class is for testing purpose.